src/OpenTelemetry.AutoInstrumentation/Configuration/ResourceConfigurator.cs

```diff
@@ -19,11 +19,11 @@
 
 namespace OpenTelemetry.AutoInstrumentation.Configuration;
 
-internal static class ResourceFactory
+internal static class ResourceConfigurator
 {
-    public static ResourceBuilder Create()
+    public static void ConfigureAutoInstrumentationResources(ResourceBuilder resourceBuilder)
```

Comment:
nit: how about renaming to `Configure`?  😉

Reply:
472f15958d4a94354f7e6977835dae811baa3007